app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/auth/LoginViewModel.kt

```diff
@@ -46,7 +42,7 @@ class LoginViewModel(
 
     fun onLoginClicked() {
         Timber.v("[onLoginClicked]")
-        viewModelScope.launch(dispatcherProvider.IO) {
+        launchIO {
```

Comment:
Very nice! Love the simpler approach with less keystrokes/faster autocomplete :)